web-quiz-app/src/components/TextQuestion.jsx

```diff
@@ -87,14 +88,26 @@ function TextQuestion() {
             question_text: question,
             question_total_marks: Number(totalMarks),
             quiz_id: quizID,
+            question_solution: solution,

```

Comment:
Note (I can't comment on the above line) 

The question_data for textual questions should be 0